### ISSUE TITLE
"Reminder Email" value doesn't take

### ISSUE THREAD

**Issue by phptek** [symbiote/silverstripe-advancedworkflow#347]
mod-php7.1 / 5.0.x-dev / framework 4.0.x

The "Reminder Email" field does not take / save when a value is inputted. This is becuase of a field mis-match in `WorkflowDefinition`.

```
    private static $db = array(
...
        'RemindDays'        => 'Int',
```

```

        if (class_exists(AbstractQueuedJob::class)) {
            $fields->addFieldToTab(
                'Root.Main',
                NumericField::create(
                    'ReminderEmail', // <-- FAIL
```

**Comment by robbieaverill** [comment]
@phptek feel like making a PR?

**Comment by robbieaverill** [comment]
Fixed with #367
